src/sdl_ui.cpp

```diff
@@ -786,6 +786,7 @@ void SdlUi::ProcessKeyDownEvent(SDL_Event &evnt) {
 			return;
 		}
 
+		[[gnu::fallthrough]];
```

Comment:
in GCC6 says:

`warning: attributes at the beginning of statement are ignored [-Wattributes]`